src/sentry/conf/server.py

```diff
@@ -1181,6 +1182,8 @@ def create_partitioned_queues(name):
     "projects:rate-limits": True,
     # Enable functionality to trigger service hooks upon event ingestion.
     "projects:servicehooks": False,
+    # Enable suspect resolutions feature
+    "projects:suspect-resolutions": True,
```

Comment:
This should be `False` by default, otherwise it will be enabled in single tenant and self hosted.